frontend/user/templates/user.html

```diff
@@ -0,0 +1,43 @@
+{% extends "base.html" %}
+{%- from 'govuk_frontend_jinja/components/inset-text/macro.html' import govukInsetText -%}
+{%- from "govuk_frontend_jinja/components/button/macro.html" import govukButton -%}
+
+{% block content %}
+  <div>
+      <h1 class="govuk-heading-xl">Apply for Funding</h1>
+  </div>
+<div>
+    {% if logged_in_user %}
+    <p class="govuk-body">
+        You are currently logged in as:
+        {% if logged_in_user.full_name %}
+            <br /><strong>{{ logged_in_user.full_name }}</strong>
+            <br /><i>({{ logged_in_user.email }})</i>
+        {% else %}
+            <br /><strong>{{ logged_in_user.email }}</strong>
+        {% endif %}
+    </p>
+    {% if logged_in_user.roles %}v
+        <p class="govuk-body">
+            <strong>You have been given the following role permissions:</strong>
+            {% for role in logged_in_user.roles %}
+            <br />{{ role }}
+            {% endfor %}
+        </p>
```

Comment:
just a thought: for fs-1882 could we add an `else` here and ask to contact us or display the appropriate error if the user doesnt have the right roles? 🤔

Reply:
Have removed section pending separate error message ticket...